building_energy_standards_data/database_files/hvac_minimum_requirement_variable_refrigerant_flow_systems_90_1.json

```diff
@@ -0,0 +1,4429 @@
+        "heating_type": null,
+        "subcategory": "VRF multisplit system",
+        "minimum_capacity": 135000,
+        "maximum_capacity": 9999999999,
+        "start_date": "9/9/1919",
+        "end_date": "9/9/2999",
+        "minimum_seasonal_energy_efficiency_ratio": null,
+        "minimum_seasonal_energy_efficiency_ratio_2": null,
+        "minimum_energy_efficiency_ratio": null,
+        "minimum_energy_efficiency_ratio_2": null,
+        "minimum_integrated_part_load_value": null,
+        "minimum_heating_seasonal_performance_factor": null,
+        "minimum_heating_seasonal_performance_factor_2": null,
+        "minimum_coefficient_of_performance_heating": 2.8,
+        "annotation": "From ASHRAE 90.1-2019 Table 6.8.1-8"
+    },
+    {
+        "template": "90.1-2016",
+        "equipment_type": "AirCooled",
+        "heating_type": "Any",
+        "subcategory": "VRF multisplit system",
+        "minimum_capacity": 0,
+        "maximum_capacity": 64999.99,
+        "start_date": "9/9/1919",
+        "end_date": "9/9/2999",
+        "minimum_seasonal_energy_efficiency_ratio": 13,
+        "minimum_seasonal_energy_efficiency_ratio_2": null,
+        "minimum_energy_efficiency_ratio": null,
+        "minimum_energy_efficiency_ratio_2": null,
+        "minimum_integrated_part_load_value": null,
+        "minimum_heating_seasonal_performance_factor": null,
+        "minimum_heating_seasonal_performance_factor_2": null,
+        "minimum_coefficient_of_performance_heating": null,
+        "annotation": "From ASHRAE 90.1-2016 Table 6.8.1-9"
+    },
+    {
+        "template": "90.1-2016",
+        "equipment_type": "AirCooled",
+        "heating_type": "Electric Resistance or None",
+        "subcategory": "VRF multisplit system",
+        "minimum_capacity": 65000,
+        "maximum_capacity": 134999.99,
+        "start_date": "9/9/1919",
+        "end_date": "12/31/2016",
+        "minimum_seasonal_energy_efficiency_ratio": null,
+        "minimum_seasonal_energy_efficiency_ratio_2": null,
+        "minimum_energy_efficiency_ratio": 11.2,
+        "minimum_energy_efficiency_ratio_2": null,
+        "minimum_integrated_part_load_value": 13.1,
+        "minimum_heating_seasonal_performance_factor": null,
+        "minimum_heating_seasonal_performance_factor_2": null,
+        "minimum_coefficient_of_performance_heating": null,
+        "annotation": "From ASHRAE 90.1-2016 Table 6.8.1-9"
+    },
+    {
+        "template": "90.1-2016",
+        "equipment_type": "AirCooled",
+        "heating_type": "Electric Resistance or None",
+        "subcategory": "VRF multisplit system",
+        "minimum_capacity": 65000,
+        "maximum_capacity": 134999.99,
+        "start_date": "1/1/2017",
+        "end_date": "9/9/2999",
+        "minimum_seasonal_energy_efficiency_ratio": null,
+        "minimum_seasonal_energy_efficiency_ratio_2": null,
+        "minimum_energy_efficiency_ratio": 11.2,
```

Comment:
@lymereJ Just wondering if this is similar to the AC requirements were we considered EER and IEER before but only IEER after. This applies to 90.1-2016 and 2010.

Reply:
I don't think so, the language is pretty clear there, here's an example:
![image](https://github.com/pnnl/building-energy-standards-data/assets/5149279/73803490-122e-413f-84c5-89d651fd9517)

Reply:
This is from 2016, and it is not as clear as the one you showed
![image](https://github.com/pnnl/building-energy-standards-data/assets/121573402/54a91204-8fe5-4477-9161-4ca742e306cc)